builds/assets/lang/enAU.json

```diff
@@ -280,6 +280,9 @@
   "menuLogoutMessage": "You were logged out.",
 
   "menuProfileTitle": "Profile",
+  "menuUserNameLabel": "userName",
```

Comment:
Should these be `"User Name"` and `"Real Name"`?

Reply:
I think you are right. I'll fix it soon.

Reply:
Fixed.